L1Trigger/DTTriggerPhase2/src/MPQualityEnhancerFilterBayes.cc

```diff
@@ -0,0 +1,316 @@
+#include "L1Trigger/DTTriggerPhase2/interface/MPQualityEnhancerFilterBayes.h"
+#include "FWCore/MessageLogger/interface/MessageLogger.h"
+
+using namespace edm;
+using namespace std;
+using namespace cmsdt;
+;
+
+// ============================================================================
+// Constructors and destructor
+// ============================================================================
+MPQualityEnhancerFilterBayes::MPQualityEnhancerFilterBayes(const ParameterSet &pset) : MPFilter(pset) {
+  // Obtention of parameters
+  debug_ = pset.getUntrackedParameter<bool>("debug");
+}
+
+MPQualityEnhancerFilterBayes::~MPQualityEnhancerFilterBayes() {}
+
+// ============================================================================
+// Main methods (initialise, run, finish)
+// ============================================================================
+void MPQualityEnhancerFilterBayes::initialise(const edm::EventSetup &iEventSetup) {}
+
+void MPQualityEnhancerFilterBayes::run(edm::Event &iEvent,
+                                       const edm::EventSetup &iEventSetup,
+                                       std::vector<metaPrimitive> &inMPaths,
+                                       std::vector<metaPrimitive> &outMPaths) {
+  // std::vector<metaPrimitive> buff;
+  // std::vector<metaPrimitive> buff2;
+
+  filterCousins(inMPaths, outMPaths);
+  if (debug_) {
+    LogDebug("MPQualityEnhancerFilterBayes") << "Ended Cousins Filter. The final primitives before Refiltering are: ";
+    for (unsigned int i = 0; i < outMPaths.size(); i++) {
+      printmP(outMPaths[i]);
+    }
+    LogDebug("MPQualityEnhancerFilterBayes") << "Total Primitives = " << outMPaths.size();
+  }
+  // refilteringCousins(buff, buff2);
```

Comment:
Is this commented out code needed? If so, please add another line of comment which explains in which case you plan to uncomment it. Otherwise just remove.